assets/component-product-info-drawer-tab.css

```
@@ -0,0 +1,151 @@
+/* Drawer */
+.product__info-drawer-trigger-wrapper {
+}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)